doc/usage/extensions/autosummary.rst

```diff
@@ -74,13 +74,22 @@ The :mod:`sphinx.ext.autosummary` extension does this in two parts:
 
      The ``toctree`` option also signals to the :program:`sphinx-autogen` script
      that stub pages should be generated for the entries listed in this
-     directive.  The option accepts a directory name as an argument;
+     directive.
+
+     The option accepts a directory name as an argument;
      :program:`sphinx-autogen` will by default place its output in this
      directory. If no argument is given, output is placed in the same directory
-     as the file that contains the directive.
+     as the file that contains the directive.  The directory is interpreted as
+     relative to the directory of the source file, unless it begins with the
+     system path separator (``/`` for Unix-likes), in which case it is relative
+     to the documentation root.
 
      You can also use ``caption`` option to give a caption to the toctree.
 
+     .. versionchanged:: 7.3
```

Comment:
Since it's a feature, I think it's more a ``versionadded`` that is needed rather than a ``versionchanged``.

Reply:
Sure, I wasn't sure how Sphinx preferred these things.  Done in 62b27c495.

Reply:
Ah no worry. Actually there isn't any strong convention. It's just that I think "versionchanged" should be reserved to "orthogonal" things changing wrt to the previous implementation without being deprecated. But anyone is free to argue !